dbt_project.yml

```diff
@@ -24,46 +24,95 @@ clean-targets:
   - "dbt_packages"
 
 vars:
-  surrogate_key_treat_nulls_as_empty_strings: true # turn on legacy behavior
+  snowplow_media_player:
+    surrogate_key_treat_nulls_as_empty_strings: true # turn on legacy behavior
+
+    # Sources
+    # snowplow__atomic_schema: 'atomic' # Only set if not using 'atomic' schema for Snowplow events data
+    # snowplow__database:  # Only set if not using target.database for Snowplow events data -- WILL BE IGNORED FOR DATABRICKS
+    snowplow__events: "{{ source('atomic', 'events') }}"
+    snowplow__media_event_names: ['media_player_event']
+
+    snowplow__percent_progress_boundaries: [10, 25, 50, 75]
+    snowplow__valid_play_sec: 30
+    snowplow__complete_play_rate: 0.99
+    snowplow__max_media_pv_window: 10
+    # please set any of the below three variables to true if the related context schemas are enabled for your warehouse, please note it cannot be used to filter the data:
+    # set to true if the YouTube context schema is enabled
+    snowplow__enable_youtube: false
+    # set to true if the HTML5 media element context schema is enabled
+    snowplow__enable_whatwg_media: false
+    # set to true if the HTML5 video element context schema is enabled
+    snowplow__enable_whatwg_video: false
+    snowplow__media_player_event_context: "com_snowplowanalytics_snowplow_media_player_event_1"
+    snowplow__media_player_context: "com_snowplowanalytics_snowplow_media_player_1"
+    snowplow__youtube_context: "com_youtube_youtube_1"
+    snowplow__html5_media_element_context: "org_whatwg_media_element_1"
+    snowplow__html5_video_element_context: "org_whatwg_video_element_1"
```

Comment:
I tried to be more consistent by using single quotes everywhere and double quotes only if nested within other strings, but I don't actually know what is the convention that we want to follow 😅 I think it would be better to fix this with a linter everywhere.

Reply:
Contributions are welcome :p but yeah, I more just meant specifically in the context names but you did everywhere which is great anyway!